server/routes/api/bookmark.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import BookmarkController from '../../controllers/BookmarkController';
+import AuthMiddleware from '../../middlewares/AuthMiddleware';
+
+const bookmarkRoutes = Router();
+const path = '/articles/:slug/bookmark';
+
+bookmarkRoutes.post(path, AuthMiddleware.checkIfUserIsAuthenticated, BookmarkController.create);
```

Comment:
Route definition doesn't seem to be in-line with how other routes have been defined and agreed by the team. Any reason for this decision?

Reply:
Yeah. Just following DRY principles. Will change though. Thanks for noticing